tools/deployment/telemetry_server/README.md

```
@@ -0,0 +1,13 @@
+# CLP Telemetry Server
+
+This directory contains the Docker Compose stack for the CLP server-side telemetry infrastructure.
+
+## Stack Components
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)